stdlib/socket/src/main/ballerina/src/socket/Module.md

```diff
@@ -1,154 +1,28 @@
-## Module overview
+This module provides an implementation for connecting to a remote socket server or acts as a server for an incoming socket request. The module facilitates three types of endpoints called `socket:Client`, `socket:UdpClient`, and `socket:Listener`.
+#### TCP Client endpoints
```

Comment:
Drop the use of the term endpoint.

Reply:
Applies to other places as well, where clients and listeners are referred to as endpoints